packages/api/src/routes/medical/internal-patient.ts

```diff
@@ -607,6 +624,7 @@ router.get(
  */
 router.post(
   "/trigger-update",
+  requestLogger,
```

Comment:
I know this was not in the scope of this PR, but this route is of no use for us anymore - it was a one-time action back when we first added address coordinates. 

If you end up sending another commit and could delete it and any reference to it, that would be good

Reply:
Ill remove it the logger but leave the endpoint for now